src/ScriptEditor/ui/Toolbar.tsx

```diff
@@ -40,8 +40,13 @@ export function Toolbar({ editor, onSave }: IProps) {
   const { ram, ramEntries, isUpdatingRAM, options, saveOptions } = useScriptEditorContext();
 
   const onOptionChange: OptionsModalProps["onOptionChange"] = (option, value) => {
-    saveOptions({ ...options, [option]: value });
-    editor?.updateOptions(options);
```

Comment:
@Snarling there was a minor bug in a previous PR - instead of applying new options, the previous ones were applied.
this PR should fix it